lib/client-assets.php

```diff
@@ -440,6 +440,19 @@ function gutenberg_register_packages_styles( $styles ) {
 }
 add_action( 'wp_default_styles', 'gutenberg_register_packages_styles' );
 
+/**
+ * Gets the stylesheet handle for the current theme.
+ *
+ * @since 6.3
+ *
+ * retrn string
+ */
+function gutenberg_get_stylesheet_handle() {
+	$stylesheet_array = explode( '/', get_option( 'stylesheet' ) );
+	$stylesheet_slug = end( $stylesheet_array );
```

Comment:
```suggestion
	$stylesheet_array = explode( '/', get_option( 'stylesheet' ) );
	$stylesheet_slug  = end( $stylesheet_array );
```

Tweak to keep linter happy.